backend/workflow_manager/workflow_v2/execution.py

```diff
@@ -116,6 +115,11 @@ def create_workflow_execution(
         execution_id: Optional[str] = None,
         mode: tuple[str, str] = WorkflowExecution.Mode.INSTANT,
     ) -> WorkflowExecution:
+        # Validating with existing execution
+        existing_execution = cls.get_execution_instance_by_id(execution_id)
```

Comment:
@vishnuszipstack @muhammad-ali-e `execution_id` can be `None` but the `get_execution_instance_by_id()` always expects a `str`. Please ensure that we don't end up with `NoneType` attribute errors

Reply:
@chandrasekharan-zipstack I think in the case of None, the query itself return None.Yea we can make the parameter as optional. Not required in this PR.